CDCgov/prime-reportstream
# Add AKDOH value to AK-DOH message

This PR Add AKDOH value to AK-DOH message

Test Steps:
1 . Download the following input file example and run the following command.
./prime fhirdata --input-file SR-HL7-AK-SR-TRANSFORMED.fhir --output-format HL7 -r metadata/hl7_mapping/receivers/STLTs/AK/AK-receiver-transform.yml --output-file SR-HL7-AK.hl7
2. Examine the MSH-6 field, it should contain the value of "AKDOH" as following screenshot
3. 
![image](https://github.com/CDCgov/prime-reportstream/assets/20068283/ce6eeff1-74c9-4bc9-858a-82bd78c2ba5f)

[SR-HL7-AK-SR-TRANSFORMED.txt](https://github.com/CDCgov/prime-reportstream/files/13664938/SR-HL7-AK-SR-TRANSFORMED.txt)


## Changes
- *Include a comprehensive list of changes in this PR*
- *(For web UI changes) Include screenshots/video of changes*

## Checklist

### Testing

### Process

## Linked Issues
- Fixes #issue
- https://app.zenhub.com/workspaces/engagement-6166edbd409257001e09f1b2/issues/gh/cdcgov/prime-reportstream/12630

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?
